packages/observability/mock-data/generate-mock-data.sh

```
@@ -0,0 +1,356 @@
+        for bucket in "${LATENCY_BUCKETS[@]}"; do
+            # Exponential-like distribution
+            if (( $(echo "$bucket < 0.1" | bc -l) )); then
+                bucket_prob=0.05
+            elif (( $(echo "$bucket < 0.5" | bc -l) )); then
+                bucket_prob=0.7
+            elif (( $(echo "$bucket < 2" | bc -l) )); then
+                bucket_prob=0.9
+            else
+                bucket_prob=0.99
+            fi
+            cumulative=$(generate_counter $(awk "BEGIN {print 1.5 * $bucket_prob}") $ELAPSED 10)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)